libs/orders/src/lib/components/order-list/order-view-dialog.spec.tsx

```diff
@@ -0,0 +1,196 @@
+import { act, render, screen } from '@testing-library/react';
+import { OrderViewDialog } from './order-view-dialog';
+import type { Order } from '../order-data-provider';
+import { BrowserRouter } from 'react-router-dom';
+import {
+  MarketState,
+  MarketTradingMode,
+  OrderStatus,
+  OrderTimeInForce,
+  OrderType,
+  Side,
+} from '@vegaprotocol/types';
+
+jest.mock('@vegaprotocol/react-helpers', () => ({
```

Comment:
Why do we need to mock here?

Reply:
for the `useCopyTimeout` hook

Reply:
otherwise the test doesn't run?

Reply:
Runs fine for me. Could you double check